src/deployment.jl

```diff
@@ -1,41 +1,57 @@
 """
-    deployment_area(n_corals::Int64, max_n_corals::Int64, density::Float64, target_areas::Vector{Float64})::Tuple{Float64,Float64}
+    version_check()::Vector{Int64}
+
+Check RME version
+"""
+function version_check()::Vector{Int64}
+    rme_ver = @RME version()::Cstring
+    return parse.(Int64, split(rme_ver, '.'))
+end
+
+"""
+    deployment_area(n_corals::Int64, max_n_corals::Int64, density::Union{Float64, Vector{Float64}}, target_areas::Vector{Float64})::Tuple{Float64,Float64}
 
 Determine deployment area for the expected number of corals to be deployed.
 
 # Arguments
 - `n_corals` : Number of corals,
 - `max_n_corals` : Expected maximum deployment effort (total number of corals in intervention set)
-- `density` : Stocking density per m²
+- `density` : Stocking density per m². In RME versions higher than 1.0.28 density needs to be a vector
+    with each element representing density per species
 - `target_areas` : Available area at target location(s)
 
 # Returns
 Tuple
 - Percent area of deployment
 - modified stocking density [currently no modifications are made]
 """
-function deployment_area(n_corals::Int64, max_n_corals::Int64, density::Float64, target_areas::Vector{Float64})::Tuple{Float64,Float64}
+function deployment_area(
+    n_corals::Int64,
+    max_n_corals::Int64,
+    density::Union{Float64,Vector{Float64}},
+    target_areas::Vector{Float64}
+)::Union{Tuple{Float64,Float64},Tuple{Float64,Vector{Float64}}}
     req_area = area_needed(max_n_corals, density)
 
     # Divide by half (i.e., `* 0.5`) as RME simulates two deployments per year
-    mod_density = (n_corals * 0.5) / (req_area / m2_TO_km2)
-    deployment_area_pct = min((req_area / sum(target_areas)) * 100.0, 100.0)
+    mod_density = (n_corals * 0.5) ./ (req_area ./ m2_TO_km2)
+    deployment_area_pct = min((sum(req_area) / sum(target_areas)) * 100.0, 100.0)
```

Comment:
Calculate `sum(req_area)` once beforehand and replace the repeated `sum()` here and below please.